File: sample/src/main/java/com/sscl/basesample/activities/sample/ImageLoaderActivity.java
package com.sscl.basesample.activities.sample;

import android.view.Menu;
import android.view.MenuItem;
import android.widget.ImageView;

import androidx.annotation.NonNull;

import com.sscl.baselibrary.activity.BaseAppCompatActivity;
import com.sscl.baselibrary.image.ImageLoader;
import com.sscl.basesample.R;

/**
 * @author jackie
 */
public class ImageLoaderActivity extends BaseAppCompatActivity {

    private static final String URL = "https://ywmdata.mohoo.club/upload/wxcode_ywm863412042671753.png";

    private ImageView imageView;

    @Override
    protected boolean titleBackClicked() {
        return false;
    }

    @Override
    protected void doBeforeSetLayout() {

    }

    @Override
    protected int setLayout() {
        return R.layout.com_sscl_basesample_activity_image_loader;
    }

    @Override
    protected void doBeforeInitOthers() {

    }

    @Override
    protected void initViews() {
        imageView = findViewById(R.id.image);
    }

    @Override
    protected void initViewData() {
        ImageLoader.getInstance(getApplicationContext()).displayImage(URL, imageView, false);
    }

    @Override
    protected void initOtherData() {

    }

    @Override
    protected void initEvents() {

    }

    @Override
    protected void doAfterAll() {

    }

    @Override
    protected boolean createOptionsMenu(@NonNull Menu menu) {
        return false;
    }

    @Override
    protected boolean optionsItemSelected(@NonNull MenuItem item) {
        return false;
    }
}

File: sample/src/main/java/com/sscl/basesample/activities/sample/ImageLoaderActivity.kt
package com.sscl.basesample.activities.sample

import android.view.Menu
import android.view.MenuItem
import android.widget.ImageView
import com.sscl.baselibrary.activity.BaseAppCompatActivity
import com.sscl.baselibrary.image.ImageLoader
import com.sscl.basesample.R

/**
 * @author jackie
 */
class ImageLoaderActivity : BaseAppCompatActivity() {
    private lateinit var imageView: ImageView
    override fun titleBackClicked(): Boolean {
        return false
    }

    override fun doBeforeSetLayout() {}
    override fun setLayout(): Int {
        return R.layout.com_sscl_basesample_activity_image_loader
    }

    override fun doBeforeInitOthers() {}
    override fun initViews() {
        imageView = findViewById(R.id.image)
    }

    override fun initViewData() {
        ImageLoader.getInstance(applicationContext).displayImage(URL, imageView, false)
    }

    override fun initOtherData() {}
    override fun initEvents() {}
    override fun doAfterAll() {}
    override fun createOptionsMenu(menu: Menu): Boolean {
        return false
    }

    override fun optionsItemSelected(item: MenuItem): Boolean {
        return false
    }

    companion object {
        private const val URL = "http://img.sccnn.com/bimg/342/16609.jpg"
    }
}